data/binding/binditems.go

```diff
@@ -39,35 +45,50 @@ func (b *boundBool) Get() bool {
 }
 
 func (b *boundBool) Set(val bool) {
-	if *b.val == val {
-		return
-	}
 	if b.val == nil { // was not initialized with a blank value, recover
```

Comment:
Can this ever be nil?

Reply:
Hmm, maybe I added more safety since this

Reply:
Yeah good catch thanks. At some point nil was possible but that loophole was closed and the code not removed.